packages/x-license-pro/src/useLicenseVerifier/useLicenseVerifier.ts

```diff
@@ -27,7 +28,11 @@ export function useLicenseVerifier(
       return sharedLicenseStatuses[packageName]!.status;
     }
 
-    const licenseStatus = verifyLicense(releaseInfo, licenseKey);
+    const acceptedScopes: LicenseScope[] = packageName.includes('premium')
```

Comment:
This logic would have to become smarter if we release other plans

Reply:
So to be sure that I understand it - the goal here is to if you have `premium` you also get `pro` right?

Reply:
Yes
I first did a scale `const scopes = ['community', 'pro', 'premium']` and then compare the index of the current plan with the one of the license
But it would not work with more granular scopes if we have some in the future
Right now it's a very basic check so both options work fine.